### PR TITLE
Disable node modification during map data load

### PR DESCRIPTION
**Pull request by etiennehomer**

### CODE REVIEW

File: src/redux/reducer.js [hunk 1]
```diff
@@ -211,6 +211,7 @@ const initialState = {
     authenticationRouterError: null,
     showAuthenticationRouterLogin: false,
     studyUpdated: { force: 0, eventData: {} },
+    mapDataLoading: true,
```
Comment (sBouzols) [inline src/redux/reducer.js:214]: I would have set it to false by default

File: src/redux/actions.js [hunk 2]
```diff
@@ -88,12 +88,6 @@ export function mapEquipmentsCreated(
     };
 }
 
-export const LOAD_GEO_DATA_SUCCESS = 'LOAD_GEO_DATA_SUCCESS';
```
Comment (sBouzols) [inline src/redux/actions.js:91]: used anywhere ? Ok to clean this

File: src/redux/reducer.js (hunk 1 above)
Comment (etiennehomer) [reply src/redux/reducer.js]: Actually it's impossible for the user to modify the node before the load of map data => I changed the initialization value to true

File: src/redux/actions.js (hunk 2 above)
Comment (etiennehomer) [reply src/redux/actions.js:91]: Yes it's some old code. It's not used anymore for a while

File: src/components/network-map-tab.js [hunk 3]
```diff
@@ -562,19 +562,20 @@ export const NetworkMapTab = ({
                     if (!checkNodeConsistency(nodeBeforeFetch)) {
                         return;
                     }
-                    setErrorMessage(
-                        intlRef.current.formatMessage(
-                            { id: 'geoDataLoadingFail' },
-                            { studyUuid: studyUuid }
-                        )
-                    );
+                    console.info('111', error.message);
```
Comment (SlimaneAmar) [inline src/components/network-map-tab.js:565]: Remove it

File: src/components/network-map-tab.js [hunk 4]
```diff
@@ -638,15 +639,16 @@ export const NetworkMapTab = ({
                 if (!checkNodeConsistency(nodeBeforeFetch)) {
                     return;
                 }
-                setErrorMessage(
-                    intlRef.current.formatMessage(
-                        { id: 'geoDataLoadingFail' },
-                        { studyUuid: studyUuid }
-                    )
-                );
+                console.info('222', error.message);
```
Comment (SlimaneAmar) [inline src/components/network-map-tab.js:642]: Remove it

**Review by SlimaneAmar — APPROVED** [review]

File: src/components/network-map-tab.js (hunk 3 above)
Comment (etiennehomer) [reply src/components/network-map-tab.js]: done

File: src/components/network-map-tab.js (hunk 4 above)
Comment (etiennehomer) [reply src/components/network-map-tab.js]: done

**Review by sBouzols — APPROVED** [review]

**Pull request merged**